src/parse.y

```diff
@@ -296,8 +297,10 @@ static void build_global(ID* id, Type* t, InitClass ic, Expr* e, std::vector<Att
 
     add_global(id_ptr, std::move(t_ptr), ic, e_ptr, std::move(attrs_ptr), dt);
 
-    if ( dt == VAR_REDEF )
+    if ( dt == VAR_REDEF ) {
+        DBG_LOG(DBG_ZEEKYGEN, "redef of %s in %s", id->Name(), ::filename);
```

Comment:
Maybe move the logging into `Redef()` directly, so it's in `src/zeekygen` ?

Reply:
That was some left over debugging when I was tracing things. I removed it.